docs/DeveloperGuide.adoc

```diff
@@ -111,6 +111,19 @@ image::DeleteSequenceDiagram.png[]
 
 NOTE: The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
 
+.Interactions inside the Logic Component for the `enqueue 001A` Command
+image::EnqueueSequenceDiagram.png[]
+
+.Interactions inside the Logic Component for the `next 1` Command
+image::NextSequenceDiagram.png[]
```

Comment:
Do you want to color out the lifeline for the NextCommandParser below the destroy marker so that the diagram is accurate or add a note to acknowledge the limitation of PlantUML like you did for the other diagrams?